ome/provider.go

```diff
@@ -268,6 +276,13 @@ func (p *omeProvider) Schema(ctx context.Context, _ provider.SchemaRequest, resp
 					fmt.Sprintf(" Default value is '%d'.", defaultTimeoutInSeconds),
 				Optional: true,
 			},
+			"https": schema.StringAttribute{
```

Comment:
Can name this field `protocol` with a default of `https` and validators to check that only two values are accepted: `http` and `https`. Please share your thoughts.

Reply:
Made the update